manifests/params.pp

```diff
@@ -29,6 +28,7 @@
 
       $path_separator = ':'
     }
+    'windows' : {    }
```

Comment:
You will need to configure all of the folder settings. Not sure if that is this PR, but I think it will be needed to finish closing out PUP-4479.